specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/mysql.json

```diff
@@ -1517,6 +1517,114 @@
         "modelAsString": false
       }
     },
+    "MinimalTlsVersion": {
+      "type": "string",
+      "description": "Enforce a minimal Tls version for the server.",
+      "enum": [
```

Comment:
Allowed value includes "TLSEnforcementDisabled" as well.

Reply:
Incorporated this